index.html

```diff
@@ -71,7 +71,7 @@
     <div class="content-center">
       <div class="container">
         <div class="title-brand" style="max-width: 960px">
-          <h1 class="presentation-title" style="font-size: 7em">Shirakami Fubuki</h1>
+          <span class="presentation-title">Shirakami Fubuki</h1>
```

Comment:
Tag has not been closed properly.

```suggestion
          <span class="presentation-title">Shirakami Fubuki</span>
```

Reply:
Whoops. I'll accept this immediately.